packages/styles/components/_c.fields.scss

```diff
@@ -0,0 +1,37 @@
+.mc-fields {
+  font-family: 'LeroyMerlin', sans-serif;
+
+  &__label {
+    @include set-font-scale('04');
+
+    color: $color-fields-label;
+    margin-bottom: $mu025;
```

Comment:
For the margin to be applied, you need a `inline-block` element (default label is `inline`). 
But I think we should better look into the spacing global strategy here. 

If I look at the design files, the spacing between le input and the text on top (either the label or the suggestion) is always `$mu050`. I think it is more relevant to do it like this : 

```scss
.mc-field {
  // ...

  &__label {
    @include set-font-scale('04', 'xs'); // set the line-height close to 1em
    // ...
  }

  &__required {
    @include set-font-scale('02', 'xs');
  }

  &__suggestion {
    @include set-font-scale('02', 'xs'); // set the line-height close to 1em
    margin-top: $mu025; // add 4 px between suggestion and label
    // ...
  }

  &__input {
    margin-top: $mu050; // add 8px between input and any text on top
    // ...
  }

  // ...
}

```